### PR TITLE
Enable 2 tests out of 6 in hamming_test.py

### PR DESCRIPTION
**Pull request by gabrieldemarmiesse**
Related to #1200 but does not fix it. It's just enable some correct tests in the CI

### CODE REVIEW

File: tensorflow_addons/metrics/hamming_test.py [hunk 1]
```diff
@@ -39,6 +40,7 @@ def update_obj_states(self, obj, actuals, preds):
     def check_results(self, obj, value):
         self.assertAllClose(value, self.evaluate(obj.result()), atol=1e-5)
 
+    @unittest.skip("Failing. See https://github.com/tensorflow/addons/issues/1200")
```
Comment (seanpmorgan) [inline tensorflow_addons/metrics/hamming_test.py]: @gabrieldemarmiesse  Just wondering what benefits we see for these PR's to use unittest.skip vs. the self method:
https://github.com/tensorflow/addons/blob/master/tensorflow_addons/image/resampler_ops_test.py#L100

Don't really have a strong preference for one over the other, but it does prevent an additional import

Comment (gabrieldemarmiesse) [reply tensorflow_addons/metrics/hamming_test.py]: I don't have a strong preference either. I think they're equivalent. Do you want me to change them to `self.skip`?

Comment (seanpmorgan) [reply tensorflow_addons/metrics/hamming_test.py]: If you don't mind it'll be consistent with the other skips we've implemented

Comment (gabrieldemarmiesse) [reply tensorflow_addons/metrics/hamming_test.py]: Sure thing

**Pull request merged**